changelog/@unreleased/pr-4607.v2.yml

```diff
@@ -0,0 +1,28 @@
+type: fix
+fix:
+  description: |-
+    SweepStrategyManagers only load one table metadata on cache miss
```

Comment:
please fix up the changelog

Reply:
What fixing up do you want? What's the usual Atlas formatting for these changelogs?

Reply:
Just a short description of the change akin to a commit message if you will. This has copied the entire PR description which is usually just bloat.